scripts/helm/install.sh

```diff
@@ -165,7 +165,10 @@ else
        --set="etcd.livenessProbe.initialDelaySeconds=5,etcd.readinessProbe.initialDelaySeconds=5,etcd.replicaCount=1" \
        --set="obs.callhome.enabled=true,obs.callhome.sendReport=false,localpv-provisioner.analytics.enabled=false" \
        --set="eventing.enabled=true,nats.cluster.enabled=false,nats.cluster.replicas=1" \
+       --set="loki.loki.commonConfig.replication_factor=2,loki.singleBinary.replicas=2,loki.minio.replicas=2" \
+       --set="loki-stack.enabled=false" \
```

Comment:
loki-stack doesn't exist anymore?

Reply:
This will disable loki-stack on the old version, thus on upgrade there will be no conflict.

Reply:
That's the reason I have added the comment below.